src/nested_pandas/nestedframe/core.py

```diff
@@ -974,6 +974,94 @@ def describe(self, exclude_nest: bool = False, percentiles=None, include=None, e
 
         return NestedFrame(pd.concat(result, axis=1))
 
+    def explode(self, column: IndexLabel, ignore_index: bool = False):
+        """
+
+        Transform each element of a list-like base column to a row, replicating index value.
+        Or unnest a specified nested column with the other columns being replicated as part
+        of the unnest. The exploded columns will be added to the right of the rest of the frame.
+
+        Parameters
+        ----------
+        column : IndexLabel
+            Base column(s) or nested column to explode.
+            For multiple base columns, specify a non-empty list with each element be string or tuple.
```

Comment:
```suggestion
            For multiple base columns, specify a non-empty list with each element being a string or tuple.
```

When it's a tuple, what is it a tuple of?

Reply:
This is from the pandas' documentation. I have also specified that the nested column should be given as a string!